src/styles/card.scss

```diff
@@ -0,0 +1,126 @@
+.catalog {
+  box-sizing: border-box;
+  display: grid;
+  grid-gap: 46px 48px;
+  justify-content: center;
+  grid-template-columns: repeat($column-count, 200px);
+  margin: 0 auto;
+  padding: 50px 0;
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, 200px);
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  &__card {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    border-radius: 5px;
+    border: 1px solid $primary-bg-color;
+    background-color: $background-card-color;
+
+    &:hover {
+      transform: scale(1.2);
+      transition: all 300ms ease;
```

Comment:
This transition should be on the element without `:hover`.
By adding the transition only to the :hover state, what happens is that the transition applies when you start hovering, but not when you stop hovering, because at that moment the element is not in the :hover state anymore.
This creates a behavior where it is smooth to  check the item, but abrupt the moment you stop checking it.
To fix, just move `transition: all 300ms ease` inside `&__card` and outside `:hover`